### DISCUSSION TITLE
我建立一个群大家一起来把

### DISCUSSION THREAD

**Discussion by lhwin** [pengxiao-song/LaWGPT#101] [General（水群）]
爱好者可以加下面交流（本人目前中国科学院信息工程研究所工作）加不上的话 邮箱lihe@iie.ac.cn

**Comment by lhwin** [comment]
![微信图片_20231015121105](https://github.com/pengxiao-song/LaWGPT/assets/43327344/3405ba9a-abba-46c2-b1f4-529bd2bd15b3)

**Reply by guoyongzhi** [reply]
二维码过期了，建议附上你的微信二维码，加你好友拉群

**Reply by 13098806890** [reply]
![7071691638602_ pic](https://github.com/pengxiao-song/LaWGPT/assets/12133181/4134b3cf-9808-49ee-b1e2-b1d037d5f866)
请拉我进群

**Reply by viviTjwan** [reply]
![527baaf9f0d931eb3cc9f1bff866fb8](https://github.com/pengxiao-song/LaWGPT/assets/50373509/32fb8270-8d18-44ca-bb08-7e28f973d2f0)
麻烦拉一下 我

**Comment by Ch1ldman** [comment]
![微信图片_20231218184604](https://github.com/pengxiao-song/LaWGPT/assets/88812227/aee23eea-3da9-4665-9e64-806422643078)
麻烦大佬们拉我一下 感谢！
